source/production/F0.Minesweeper.Components/Game.razor.cs

```diff
@@ -0,0 +1,9 @@
+using F0.Minesweeper.Logic.Abstractions;
+
+namespace F0.Minesweeper.Components
+{
+	public partial class Game
+	{
+		public MinefieldOptions MinefieldOptions = new MinefieldOptions(10, 10, 2, MinefieldFirstUncoverBehavior.MayYieldMine);
```

Comment:
[TTNE](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/target-typed-new)? 😛

Reply:
Haha, nice catch :D It will take so much time to get accustomed to this ^^